Documentation/k8s-pre-reqs.md

```diff
@@ -67,68 +38,35 @@ Setting up kubeconfig...
 Kubectl is now configured to use the cluster.
 ```
 
-SSH into the minikube host and install `rbd`:
-
-```console
-$ minikube ssh
-$ cd /bin
-$ sudo curl -O https://raw.githubusercontent.com/ceph/ceph-docker/master/examples/kubernetes-coreos/rbd
-$ sudo chmod +x /bin/rbd
-$ rbd #run command to download ceph images.
-Unable to find image 'ceph/base:latest' locally
-latest: Pulling from ceph/base
-...
-$ exit
-```
-
 After these steps, your minikube cluster is ready to install Rook on.
 
-## Existing Kubernetes Cluster
-
-Alternatively, if you already have a running Kubernetes cluster, you can deploy Rook to it with a small update to modify the kubelet service to bind mount `/sbin/modprobe`, which allows access to `modprobe`.
-Access to modprobe is necessary for using the rbd volume plugin, which is being tracked in the Kubernetes code base with [#23924](https://github.com/kubernetes/kubernetes/issues/23924).
-
-If using RKT, you can enable `modprobe` by following this [guide](https://github.com/coreos/coreos-kubernetes/blob/master/Documentation/kubelet-wrapper.md#allow-pods-to-use-rbd-volumes). Instructions have been directly copied below for your convenience:
-
-Add the following options to the `RKT_OPTS` env before launching the kubelet via kubelet-wrapper:
-
-```ini
-[Service]
-Environment=KUBELET_VERSION=v1.5.3_coreos.0
-Environment="RKT_OPTS=--volume modprobe,kind=host,source=/usr/sbin/modprobe \
```

Comment:
The flexvolume driver gets copied to the host directory via hostpath mount. So it doesnt matter whether kubelet is containerized or not.  This was needed before because kubelet was running `modprobe` via the rbd plugin, whose code lives in kubelet.

Reply:
But one thing that you should done is for containerized kubelet to provide docker host mount for the flexvolume directory. Im assuming this is done by the coreos kubelet service wrapper file. If not, it is a bug.